database/metadata/databases/default/tables/public_user_events.yaml

```diff
@@ -27,20 +27,3 @@ insert_permissions:
         - event_name
         - user_email
         - created_at
-select_permissions:
```

Comment:
This is following up on [a note](https://github.com/cityofaustin/vision-zero/pull/1906#discussion_r2607405398) from @johnclary in the previous PR.

I dropped `SELECT` permissions for the editor and vz-admin roles. Hasura doesn't make the admin role editable in the hasura console permissions UI or the metadata tables so we could still build out some kind of Analytics dashboard page only visible for admins with the permissions as they stand now. 

<img width="1194" height="498" alt="Image" src="https://github.com/user-attachments/assets/dac04c59-fc33-4bfd-95fc-38fd4dec472e" />